torch/linalg/__init__.py

```diff
@@ -140,3 +140,33 @@
     >>> LA.norm(m[0, :, :]), LA.norm(m[1, :, :])
     (tensor(3.7417), tensor(11.2250))
 """)
+
+tensorsolve = _add_docstr(_linalg.linalg_tensorsolve, r"""
+linalg.tensorsolve(a, b, axes=None) -> Tensor
+
+Computes the solution to the tensor equation ``a x = b`` for ``x``.
+
+Suppose that `:attr:`b` is equivalent to ``torch.tensordot(a, x)``.
+This function computes tensor ``x`` from `:attr:`a` and `:attr:`b`.
+
+Args:
+    a (Tensor): left-hand-side tensor of size ``b.shape + Q``.
```

Comment:
I'm not sure what this is trying to say. I like that it's trying to describe the input shape but I'm not following the description.

Reply:
Maybe the confusion comes from using the word "size", I don't know what is the preferred term now, but I was referring to the shape of the tensor here (`tensor.size() == tensor.shape`).
`other.shape` is a tuple in NumPy and `torch.Size` in PyTorch and `Q` is a tuple, then `input.shape == other.shape + Q`.
I should change `Q`->`x.shape`.